docs/source/azure.rst

```diff
@@ -54,6 +54,42 @@ definition, the following options are available.
 
 ⚫ If you declare both public and private image, only the private will be taken
 
+azure_api
+-------
+
+Any Azure resources can be provisioned using this role, it supported by the Azure Api
+
+* Example <workspaces/azure/Pinfile>`
+* azure_api module <https://docs.ansible.com/ansible/latest/modules/azure_rm_resource_module.html#azure-rm-resource-module>`_
+* Azure API <https://docs.microsoft.com/en-us/rest/api/compute/>`_
```

Comment:
I think a better link would be https://docs.microsoft.com/en-us/rest/api/?view=Azure